### DISCUSSION TITLE
Announcements

### DISCUSSION THREAD

**Discussion by FlareFlo** [Warthunder-Open-Source-Foundation/wt_ext_cli#2] [Announcements]
test

**Comment by naprolom4ik** [comment]
Where to put commands?

**Reply by FlareFlo** [reply]
After the command itself, its a CLI app. The usage_manual.md explains this.

**Reply by naprolom4ik** [reply]
Can you give an example of using "wt_ext_cli unpack_vromf --format BlkText -i" please?

**Reply by naprolom4ik** [reply]
"wt_ext_cli unpack_vromf --format BlkText -i C:\Users\naprolom4ik\AppData\Local\WarThunder" creates broken .bin files in WarThunder folder

**Reply by naprolom4ik** [reply]
wt_ext_cli.exe is in the game folder

**Reply by FlareFlo** [reply]
Can you try just a single vromf, when passing the entire folder it will attempt every reachable .bin file

**Reply by FlareFlo** [reply]
`wt_ext_cli unpack_vromf -i ./aces.vromfs.bin --format BlkText` should be all you need

**Comment by naprolom4ik** [comment]
![WTscreenshot](https://github.com/Warthunder-Open-Source-Foundation/wt_ext_cli/assets/148037576/dc3e791d-9d4d-4db6-b44f-7f78fcdd52f3)
That russian text says "The specified file cannot be found"

**Reply by FlareFlo** [reply]
aces path of course needs to be replaced with a valid vromf location

**Reply by naprolom4ik** [reply]
Everything worked, thanks
